samples/Azure.Management.Storage/Generated/StorageAccountContainer.cs

```diff
@@ -351,7 +351,7 @@ public AsyncPageable<StorageAccount> ListAsync(CancellationToken cancellationTok
         {
             async Task<Page<StorageAccount>> FirstPageFunc(int? pageSizeHint)
             {
-                using var scope = _clientDiagnostics.CreateScope("StorageAccountContainer.ListByResourceGroup");
+                using var scope = _clientDiagnostics.CreateScope("StorageAccountContainer.List");
```

Comment:
where is the next page?

Reply:
Next page is null here - https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json#L370